File: modul4/Python/PRAK405-2310817210029-Putra_Whyra_Pratama_Setiawan.py
hasilseluruh = 0
a, b = map(int, input("").split(" "))
for i in range(1, a + 1, +1):
    hasilbaris = 0
    for j in range(i, 0, -1):
        hasilbaris += j * b
        print(f"({j}*{b})", end="")
        if j > 1:
            print(" + ", end="")
        else:
            print(f" = {hasilbaris}")
            break
    hasilseluruh += hasilbaris
print(f"{hasilseluruh}")

File: modul4/C/PRAK405-2310817210029-Putra_Whyra_Pratama_Setiawan.c
#include <stdio.h> 
int main() {
    int a, b, i, j, hasilseluruh = 0;
    scanf("%d %d", &a, &b);
    for(i = 1; i <= a; i++) {
        int hasilbaris = 0;
        for(j = i; j >= 1; j--) {
            hasilbaris += j * b;
            printf("(%d*%d)", j, b);
            if(j > 1) {
                printf(" + ");
            } else {
                printf(" = %d", hasilbaris);
                break;
            }
        }
        hasilseluruh += hasilbaris;
        printf("\n");
    }
    printf("%d", hasilseluruh);
}